examples/RL/ppo_policy_training.ipynb

```diff
@@ -0,0 +1,387 @@
+{
```

Comment:
It got a little confusing to have a `ppo.py` and a `ppo_policy_training.ipynb` here in the same folder. I would:

- Rename `ppo.py` to another more meaningful name, what is this file doing ? Just looking at it, it looks like it is a PPO implementation, but it isn't;
- create a new folder here called "notebooks" and move this notebook to inside of it and adjust the paths

Reply:
- `ppo.py` updated to `ppo_training.py`
- new folder `notebooks` created